content/en/synthetics/guide/browser-tests-using-shadow-dom.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Using Shadow DOM In Your Browser Tests
+kind: guide
+further_reading:
+    - link: '/synthetics/browser_tests'
+      tag: 'Documentation'
+      text: 'Learn about Browser Tests'
+---
+
+## Overview
+
+The Shadow Document Object Model (DOM) API is a web component that allows an encapsulated DOM tree to be attached to an HTML element. The [shadow DOM][1] is self-contained and stays isolated from the main document's DOM. 
+
+You may use a shadow DOM for the following use cases:
+
+- Forms and components from third-party libraries
+- Embedded content (such as a video or an image)
+- Chat pop-up integrations
+
+<div class="alert alert-info">
+Because the shadow DOM is encapsulated from the HTML element, Synthetic browser tests have limited access to elements rendered in the shadow DOM. The full suite of available actions and assertions may not be supported for shadow DOM elements.
+</div>
+
+Depending on the [encapsulation mode][2] and the step objective, you can leverage browser test actions to configure a test that interacts with and validates elements rendered within a shadow DOM. 
+
+## Open mode
+
+{{< img src="synthetics/guide/browser-tests-using-shadow-dom/open-shadow-dom.png" alt="Open Shadow DOM" style="width:50%;" >}}
```

Comment:
for both Open and Close mode, I will re-state that normal assertions do not work, and 
- for Open, offer to use JS steps
- for Close, offer to use actions